### ISSUE TITLE
Minimize customization in Travis, Coveralls and Code Climate

### ISSUE THREAD

**Issue by cuducos** [dirtycoder/pets#24]
Hi all,

I've been analyzing some files and IMHO there are a lot of settings in `.travis.yml` and `.codeclimate.yml` that requires more maintenance than it should. And I would like to minimize this. Some ideas (not sure if all of them are feasible):

**`.travis.yml`**

* Less (or at least more readable _env vars_): one per line, **or** (even better) use `python-decouple` (already a dependency I guess; or even `os.environ`) to set those values as defaults in the app it self;
* Do we really need to test for 3.4 and 3.5? We should go for a specific version I guess (and be sure that it passes);
* As I might touch on database configuration for the last item, any pro or con of working with [dj-database-url](https://github.com/kennethreitz/dj-database-url)? I like it;
* `coverage` command requires us to manually add each app, there might be a better way to set this include/exclude argument that requires less maintenance.

**`.codeclimate.yml`**

* `exclude_paths` requires us to manually manage it, there might be a better way to set this that requires less maintenance;
* `languages`: do we need Ruby, JS and PHP?

I'm willing to tackle this list (if we all agree that they are valid claims) and surely take #21 into account. What do you think?

**Comment by dirtycoder** [comment]
Hey @cuducos, oh great!

So, lets see, about the `.travis.yml`:
1) As we're using `python-decouple` I think we could move some variables to be default there, or I noticed that there's a config to add envvars in the Travis web interface, my bad.
2) I believe so, the server currently runs on 3.4 but I'd like to always make sure that the code runs without problem in the next version too.
3) I can't think in any con right now, btw I like it too.
4) To be honest I didn't made much research about this, but if there's a better way, let's go for it! :)

About `.codeclimate.yml`:
1) I didn't make much research on this too, but I'm ok about moving to a better way.
2) About Ruby and PHP I think we don't need it, I probably just forgot both there. But I'd like to leave the JS, check [here](https://codeclimate.com/github/dirtycoder/pets/pets/static/js/pets.js).

For the #21, that would be awesome! Let me know if you need something.

Can I add a little more request? :P
I'd love to check for PEP8 compliance before each build.

**Comment by dirtycoder** [comment]
Just added this values to the Travis config:
![envvars](https://cloud.githubusercontent.com/assets/1988429/16714400/36ae27c8-4698-11e6-9247-9640025c0de6.PNG)

**Comment by cuducos** [comment]
Ok, great!

I'm gonna do some improvements and share. If you like the results I proceed with the other changes… 

**`.travis.yml`**

* As you are using the web panel for that, I'll leave that out of this first round, unless you ask me to use default values;
* That's a great thing to have in mind! Yay, I've learned something today ; )
* If I end up implementing some default for settings, I bring [dj-database-url](https://github.com/kennethreitz/dj-database-url) in then — in other words, same as the first point;
* Great, I'm gonna suggest something here in the next commit.

**`.codeclimate.yml`**

* Great, I'm gonna suggest something here in the next commit;
* Gonna clean up too!
